### PR TITLE
Migrate swiss to Ubuntu 18.04 machine

### PR DESCRIPTION
**Pull request by dannyroberts**
##### SUMMARY
This replaces the swiss server with a new machine that's set up on Ubuntu 18.04. It includes all the fixes that were needed along the way.

##### ENVIRONMENTS AFFECTED
swiss

### CODE REVIEW

File: environments/swiss/public.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ elastcsearch_backup_days: 2
 
 redis_maxmemory: 4gb
 
-ssh_allow_password: True
+ssh_allow_password: False
```
Comment (proteusvacuum) [inline environments/swiss/public.yml]: what's the reason for this change? I'm assuming this was enabled for someone on the swiss team to be able to access the server. just making sure they know about this.

Comment (dannyroberts) [reply environments/swiss/public.yml]: Yeah I asked Gonçalo about it and he confirmed that I could change it to use our recommended setting here

File: environments/swiss/known_hosts [hunk 2]
```diff
@@ -1,2 +1,5 @@
+185.19.29.53, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBIt7cPp2d9tUMvmiO/MBAdsUUqdfoRBjjuPkcGeA0Y+b0rNctzEoBzsiVGk9ycTZ+3IXYGkwzGnfQ46zARfCcmk=
+185.19.29.53, ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPU4gQfLOtpD0iRX+X68vK36Yi5lIdGgdJVcpQV7mXcR
+185.19.29.53, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCpgr9sH/P071YEy+4HR1dnzynRf+qTWRdzyf0ooJD3G9s8Urgpq5VtyUJ8yL4D2gzAd2CZdhymGYnoM+Ya0HzCLwkDv4PAR/O8V8J6M9pt4IO8z1wVOzhEyumrGOkdY4gFPXSnpvdcdY8B5GN1QcDmadj0CEfyhDuMZXPifrxCvwe2V4ZyybjDvSbtPsR/nHFGB/ex/PLO95XbhGYYwvvr8lXFYiilSPcUCyEh2uBgbn0smbOJlCkdLKUzd8EH38/s8PnXIEdGc8Uf6DZw3h3abAIYBtv9i2A3LY6UxBdbW9ynhbfb0ANuV8ezvHI+Ovrs+n2Sop58EdrAvSxpRqfF
 swiss.commcarehq.org,185.12.7.167 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMeXDBTYYMyzZIWoDi6MBBS0era8VS9DuP3/vn5oQSgRbFTQMj9K7OH42O2r923tWKx+8FM8QDBrSJZvIydKt3I=
```
[the remark this answers is not in the document]
Comment (dannyroberts) [reply environments/swiss/known_hosts]: I think the way to do that would be to delete this file and rebuild it from scratch. I can do that after it's all over

File: src/commcare_cloud/fab/fabfile.py [hunk 3]
```diff
@@ -286,7 +284,6 @@ def env_common():
         'pgstandby': pg_standby,
         'elasticsearch': elasticsearch,
         'riakcs': riakcs,
-        'rabbitmq': rabbitmq,
```
Comment (snopoke) [inline src/commcare_cloud/fab/fabfile.py]: why did this get removed?

Comment (dannyroberts) [reply src/commcare_cloud/fab/fabfile.py]: I'm removing it here from fab because it's unused, and because on the new swiss server we'll be using redis as the celery backend instead of rabbitmq. _That_ is in turn because of the reasons @emord noted here: https://github.com/dimagi/commcare-cloud/pull/2917.
